testutil/test_app.go

```diff
@@ -176,36 +181,43 @@ func AddGenesisAccount(addr sdk.AccAddress, appState app.GenesisState, cdc codec
 
 // GenesisStateWithSingleValidator initializes GenesisState with a single validator and genesis accounts
 // that also act as delegators.
-func GenesisStateWithSingleValidator(t *testing.T, testApp *app.App) (app.GenesisState, *tmtypes.ValidatorSet) {
-	t.Helper()
-
+func GenesisStateWithSingleValidator(testApp *app.App, genAccounts ...string) (app.GenesisState, *tmtypes.ValidatorSet, keyring.Keyring) {
```

Comment:
Why remove `t *testing.T`?

Reply:
tbh, I don't remember fully... man this PR really blew up in scope... at once point in time, I think I was using this, or some util that used this function somewhere that didn't have access, such as the testnode package, but I don't think those changes made it to this commit.

the good news is that errors that occur in this function would have done something similar when requiring that there be no errors. We should still fix this in https://github.com/celestiaorg/celestia-app/issues/1114